test/test_nn.py

```diff
@@ -3910,6 +3910,40 @@ def test_Conv2d_inconsistent_types_on_GPU_without_cudnn(self):
             # but it should work with the same type
             nn.functional.conv2d(inputs.float(), weights.float(), bias.float())
 
+    @unittest.skipIf(not TEST_CUDA, 'CUDA not available')
+    def test_Conv2d_size_1_kernel_on_GPU_without_cudnn(self, dtype=torch.float):
+        inputs = torch.randn(2, 3, 5, 5, device="cuda", dtype=dtype, requires_grad=True)
+        with cudnn.flags(enabled=True, benchmark=True, deterministic=True):
```

Comment:
You can unify your 2 tests (that look exactly the same to me) with something like:
```
kwargs = ({'enabled': True}, {'enabled': False}) 
for kw in kwargs: 
      with torch.backends.cudnn.flags(**kw): 
```